.ci/deploy-mac.sh

```diff
@@ -33,7 +35,7 @@ echo "IconIndex=0" >> Quickstart.url
 
 DMG_FILEPATH="$BUILD_ARTIFACTSTAGINGDIRECTORY/rpcs3-v${COMM_TAG}-${COMM_COUNT}-${COMM_HASH}_macos.dmg"
 
-create-dmg --volname RPCS3 \
+"/usr/local/Homebrew/bin/create-dmg" --volname RPCS3 \
```

Comment:
I don't think this path is valid. Should probably be
```suggestion
"/usr/local/bin/create-dmg" --volname RPCS3 \
```
or
```suggestion
"/usr/local/opt/create-dmg/bin/create-dmg" --volname RPCS3 \
```